CompetitionTemplate/src/competition/subsystems/drive/PoseSubsystem.java

```diff
@@ -70,6 +76,9 @@ public void updateCurrentHeading() {
         lastImuHeading = imu.getYaw();
         
         currentHeadingProp.set(currentHeading.getValue());
+        
```

Comment:
This function should be renamed to reflect the fact that it does more than update heading